applications/simulate_prod.py

```diff
@@ -0,0 +1,332 @@
+#!/usr/bin/python3
+
+"""
+    Summary
+    -------
+    This application is used to run simulations for productions (typically on the grid).
+    It allows to run a Paranal (CTAO-South) or La Palma (CTAO-North) array layout simulation
+    with the provided "prod_tag" simulation configuration (e.g., Prod6)
+    for a given primary particle, azimuth, and zenith angle.
+
+    The entire simulation chain is performed, i.e., shower simulations with CORSIKA
+    which are piped directly to sim_telarray using the sim_telarray multipipe mechanism.
+    This script assumes that all the necessary configuration files for CORISKA and
+    sim_telarray are available. FIXME - This is not true at the moment, need to fix I guess.
+    The multipipe scripts will be produced as part of this script.
+
+    This script does not provide a mechanism to submit jobs to a batch system like others
+    in gammasim-tools since it is meant to be executed on a grid node
+    (distributed to it by the workload management system).
+
+    Command line arguments
+    ----------------------
+    production_config (str, Path, required)
+        Simulation configuration file
+        (contains the default setup which can be overwritten by the command line options).
+    model_version (str, required)
+        The telescope model version to use (e.g., Prod5).
+    site (str, required)
+        North or South (case insensitive).
+    primary (str, required)
+        Name of the primary particle to simulate. The available options are
+        gamma, gamma_diffuse, electron, proton, muon, helium, nitrogen, silicon, and iron.
+    from_azimuth_direction (str, required)
+        Should be one of North, South, East, West (case insensitive).
+    zenith_angle (float, required)
```

Comment:
Why dont we have azimuth_angle as well?

Reply:
This is because the direction at the moment cannot be an angle (will explain further below).

Reply:
Sorry, I did not find the explanation below. is it the explanation in which you mention that this is planned for afterwards?

Reply:
Yes, I meant the statement about keeping things similar to the current production (no angles). Sorry, should have made it clearer here.